### PR TITLE
[chore] Prepare release v1.9.0/v0.102.0

### PR DESCRIPTION
**Pull request by opentelemetrybot**
The following commands were run to prepare this release:
- make chlog-update VERSION=v1.9.0/v0.102.0
- make prepare-release PREVIOUS_VERSION=1.8.0 RELEASE_CANDIDATE=1.9.0 MODSET=stable
- make prepare-release PREVIOUS_VERSION=0.101.0 RELEASE_CANDIDATE=0.102.0 MODSET=beta

### CODE REVIEW

**Comment by mx-psi** [comment]
update-otel PR: open-telemetry/opentelemetry-collector-contrib/pull/33342

**Review by songy23 — APPROVED** [review]

**Review by codeboten — COMMENTED** [review]
What changes caused the bump of the stable module?

**Comment by mx-psi** [comment]
@codeboten #10148, used by profiles, listed here on this PR: https://github.com/open-telemetry/opentelemetry-collector/blob/8a746978aafa23c13a9811e6ee407653658bedc8/CHANGELOG-API.md?plain=1#L24-L25 (API only change)

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,38 @@ If you are looking for developer-facing changes, check out [CHANGELOG-API.md](./
 
 <!-- next version -->
 
+## v1.9.0/v0.102.0
+
+### 🛑 Breaking changes 🛑
+
+- `exporterhelper`: remove deprecated RequestMarshaler & RequestUnmarshaler types (#10283)
+- `service`: remove deprecated Telemetry struct and New func (#10285)
+- `configtls`: remove deprecated LoadTLSConfigContext funcs (#10283)
```
Comment (mx-psi) [inline CHANGELOG.md:16]:
```suggestion
```

File: CHANGELOG-API.md [hunk 2]
```diff
@@ -7,6 +7,27 @@ If you are looking for user-facing changes, check out [CHANGELOG.md](./CHANGELOG
 
 <!-- next version -->
 
+## v1.9.0/v0.102.0
+
+### 🛑 Breaking changes 🛑
+
+- `otelcol`: Remove deprecated `ConfigProvider` field from `CollectorSettings` (#10281)
```
Comment (mx-psi) [inline CHANGELOG-API.md:14]:
```suggestion
- `otelcol`: Remove deprecated `ConfigProvider` field from `CollectorSettings` (#10281)
- `exporterhelper`: remove deprecated RequestMarshaler & RequestUnmarshaler types (#10283)
- `service`: remove deprecated Telemetry struct and New func (#10285)
- `configtls`: remove deprecated LoadTLSConfigContext funcs (#10283)
```

**Review by mx-psi — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

**Pull request merged**